packages/mask/src/extension/popups/pages/Wallet/components/WalletInfo/index.tsx

```diff
@@ -66,8 +66,9 @@ const useStyles = makeStyles()({
 export const WalletInfo = memo(() => {
     const wallet = useWallet()
     const history = useHistory()
+    const address = new URLSearchParams(useLocation().search).get('address')
 
-    const { value: domain } = useReverseAddress(wallet?.address, NetworkPluginID.PLUGIN_EVM)
+    const { value: domain } = useReverseAddress(address ?? wallet?.address, NetworkPluginID.PLUGIN_EVM)
```

Comment:
```suggestion
    const { value: domain } = useReverseAddress(address || wallet?.address, NetworkPluginID.PLUGIN_EVM)
```

`||` can circumvent `address === ''`.